config/index.js

```diff
@@ -98,19 +123,20 @@ function readRcConfig() {
  * @returns {Object}
  */
 function pickConfigFieldsFromEnvVars(configFields) {
-	return configFields.reduce((out, key) => {
-		if (ENV_VARS[key] in process.env) {
-			const val = process.env[ENV_VARS[key]];
+	return configFields.reduce((out, {name, type}) => {
+
+		if (ENV_VARS[name] in process.env) {
+			const val = process.env[ENV_VARS[name]];
 
-			switch (val) {
-				case 'true':
-					out[key] = true;
+			switch (type) {
+				case 'bool':
+					out[name] = (val === 'true');
```

Comment:
I'm curios if this wouldn't fail set-up for some people out there. E.g. from command line point of view, all of the following should yield same result:
`command --bool`
`command --bool=true`
`command --bool=1`

So, unless yargs library makes some unification there, we can't just check if it strictly equals to 'true'. Please, double-check this.

Reply:
I double checked and yargs will parse what you provided and always return you a string as the result.
Only thing to notice is that first two scenarios will work as expected, but the third one will return you a 'false' string.
I guess that is just the way they process what we'd sent them via request.
Please let me know if we need to discuss this further.

Reply:
In that case `command --bool=1` should be failing now as well, so it's fine. The main thing is that we are not breaking user's set-up with this change.